webapp-django/crashstats/crashstats/utils.py

```diff
@@ -348,6 +349,8 @@ def enhance_json_dump(dump, vcs_mappings):
 
         for frame in frames:
             enhance_frame(frame, vcs_mappings)
+            for inline in frame.get("inlines") or []:
```

Comment:
The inline structure has keys function, file, and line and uses them in the same way as the frame structure, so we can call `enhance_frame` and get the same stuff out. Namely, we want `source_link`.